src/App.js

```diff
@@ -1,163 +1,162 @@
+import FullTimeForm from "./Components/PostOpportunity/sections/FullTime/FullTime";
+import InternshipForm from "./Components/PostOpportunity/sections/Internships/Internships";
+import TechConfForm from "./Components/PostOpportunity/sections/TechConf/TechConf";
+import HackathonForm from "./Components/PostOpportunity/sections/Hackathons/Hackathons";
+import CodingCompForm from "./Components/PostOpportunity/sections/CodingComp/CodingComp";
+import ScholarshipForm from "./Components/PostOpportunity/sections/Scholarships/Scholarships";
 
-import Footer from "./Components/Footer/Footer"
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom"
+import { ViewOpportunity } from "./Components/ViewOpportunity/";
+
+import Footer from "./Components/Footer/Footer";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
 function App() {
   return (
     <Router>
-      <div className = "App">
+      <div className="App">
         <Switch>
-          <Route 
+          <Route
             exact
-            path = "/"
-            render = {(props) => (
+            path="/"
+            render={(props) => (
               <div>
                 <HomePage />
               </div>
             )}
           />
-          <Route 
+          <Route
             exact
-            path = "/viewopportunity"
-            render = {(props) => (
+            path="/viewopportunity"
+            render={(props) => (
               <div>
                 <ViewOpportunity />
               </div>
             )}
           />
-          <Route 
+          <Route
             exact
-            path = "/postopportunity"
-            render = {(props) => (
+            path="/postopportunity"
+            render={(props) => (
               <div>
                 <PostOpportunity />
               </div>
             )}
           />
-          <Route 
+          <Route
             exact
-            path = "/viewopportunity/fulltime"
-            render = {(props) => (
+            path="/viewopportunity/fulltime"
+            render={(props) => (
               <div>
-                <FullTime />
+                <ViewOpportunity path="job" />
               </div>
             )}
           />
-          <Route 
+          <Route
             exact
-            path = "/viewopportunity/hackathon"
-            render = {(props) => (
+            path="/viewopportunity/hackathon"
+            render={(props) => (
               <div>
-                <Hackathons />
+                <ViewOpportunity path="hackathon" />
               </div>
             )}
           />
-          <Route 
+          <Route
             exact
-            path = "/viewopportunity/scholarship"
-            render = {(props) => (
+            path="/viewopportunity/scholarship"
+            render={(props) => (
               <div>
-                <Scholarships />
+                <ViewOpportunity path="competition" />
```

Comment:
Shouldn't this path be scholarship? @abdus

Reply:
yes. my bad.